### PR TITLE
Update Eslint to 4.8.0 and fix greenkeeper

### PR DESCRIPTION
**Pull request by Arzte**
No more in range broken issues plz

### CODE REVIEW

**Comment by Arzte** [comment]
I like how travis fails on this

File: package.json [hunk 1]
```diff
@@ -27,10 +26,11 @@
   },
   "homepage": "https://github.com/SteamingMutt/DiscordFeedback#readme",
   "devDependencies": {
-    "eslint": "^4.7.0",
+    "eslint": "^4.8.0",
+    "ajv": "^5.2.3",
```
Comment (Dougley) [inline package.json]: ?

**Review by Dougley — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (Arzte) [reply package.json]: See the comment of the commit that adds it, it’s a new required peer dep for eslint.

Comment (Dougley) [reply package.json]: At least sort it correctly then, it's all in alphabetical order.

Comment (Arzte) [reply package.json]: 👍

**Comment by Azrael-Azzy** [comment]
Nerds.

**Review by Dougley — APPROVED** [review]

**Pull request merged**